.github/workflows/build.yml

```diff
@@ -82,66 +139,57 @@ jobs:
             goarch: arm64
           - goos: windows
             goarch: arm
+          - goos: darwin
+            goarch: 386
+          - goos: darwin
+            goarch: arm
```

Comment:
This is correct, but I did find it a little confusing that our goreleaser did not have this: https://github.com/hashicorp/terraform-provider-random/blob/532c47c4ab3fc09e9f40c896b965aab0d5be585e/.goreleaser.yml#L29-L35

Perhaps a default ignore by goreleaser for `darwin`?

Reply:
Found it -- goreleaser silently suppresses invalid os+arch pairs:

https://github.com/goreleaser/goreleaser/blob/ef10a7839c32673f16ed9455c02b1668a1fd9632/internal/builders/golang/targets.go#L250-L261